oshi-core/src/main/java/oshi/hardware/HWDiskStore.java

```diff
@@ -458,4 +459,21 @@ public boolean equals(Object obj) {
         return this.size == other.size;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public String toString() {
+        boolean readwrite = getReads() > 0 || getWrites() > 0;
+        StringBuilder sb = new StringBuilder();
+        sb.append(' ').append(getName()).append(": ");
```

Comment:
Can we leave the space on the SystemInfoTest class and start this one with the name?